Docs/source/running_cpp/parameters.rst

```diff
@@ -1507,6 +1507,24 @@ Reduced Diagnostics
     The separator between row values in the output file.
     The default separator is a whitespace.
 
+In-situ visualization
+^^^^^^^^^^^^^^^^^^^^^
+
+Besides the diagnostics described above, WarpX has in-situ visualization capabilities.
+This is controlled by the following option(s):
+
+* ``insitu.int`` (`integer`; 0 by default)
```

Comment:
As a follow-up PR, can we unify `int` and `start` with the syntax introduced by @NeilZaim in #925, please?

Reply:
As another follow-up PR, can we rename the generic `insitu.<...>` name to the more descriptive `sensei.<...>` please?

Reply:
cc @lge0303